### PR TITLE
Moves Font Awesome into source tree from jsdelivr

### PR DESCRIPTION
**Pull request by JJRcop**
For a while we've been fetching Font Awesome (the font that is used for the icons in tgui, like the eye in the top left, or the X) from the jsdelivr CDN. I've downloaded the required files, moved them into the repo, then set them up to compile into the .rsc like everything else. I also tweaked the urls in the fontawesome CSS and tgui JS as necessary to use the versions from the .rsc instead of jsdelivr.

### CODE REVIEW

**Comment by Incoming5643** [comment]
doesn't really need a changelog, it's not something a player could even notice.

**Comment by JJRcop** [comment]
M'alright

**Comment by Mervill** [comment]
Wait where did you download font awesome from? could be an opportune time to upgrade to the latest version

**Comment by JJRcop** [comment]
I downloaded it from the exact same url we've been using, we can always upgrade later but we don't really use the new icons right now. It's really simple if you want to upgrade it later.

**Comment by AnturK** [comment]
Will get rid of the random xss warning popups so 👍

**Comment by Iamgoofball** [comment]
:-1:

Store a copy locally grabbed from the webserver. I don't want to have to update this manually every time.

Also, remember licenses.

File: tgui/src/tgui.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ act(ref, 'tgui:initialize')
 
 // Load fonts.
 import { loadCSS } from 'fg-loadcss'
-loadCSS('https://cdn.jsdelivr.net/fontawesome/4.5.0/css/font-awesome.min.css')
+loadCSS('font-awesome.min.css')
```
Comment (Iamgoofball) [inline tgui/src/tgui.js]: great, now we're gonna be out of date AND violating licenses

Comment (JJRcop) [inline tgui/src/tgui.js]: we're not violating any of the licenses that govern font awesome

File: tgui/LICENSE.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+MIT license
```
Comment (Iamgoofball) [inline tgui/LICENSE.md]: but it's not MIT

Comment (JJRcop) [inline tgui/LICENSE.md]: Maybe not, but it's the MIT license
https://opensource.org/licenses/mit-license.html

**Comment by optimumtact** [comment]
Test merge

**Comment by AnturK** [comment]
This is fine, stop spreading misinformation goof.
[fa has very os friendly license](http://fontawesome.io/license/l)

**Comment by Mervill** [comment]
Here's the link: http://fontawesome.io/license/

**Pull request merged**